### ISSUE TITLE
Inline styles test should be a warning

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#231]
Since it's commonly used AND not a failure for WCAG (neither for RGAA) then it should be a warning to ensure that it's **wittingly** used.

Thanks to @GoOz who noticed it :)
